server/tests/helper/graph_helper/test_graph_helper.py

```diff
@@ -73,7 +73,6 @@ def setUpClass(cls) -> None:
                 )
             )
         )
-        TestGraphHelper.planner_response_model.set_plan_hashes()
```

Comment:
Removed all explicit calls to hash plans because the model validator runs automatically after assignment to a PlanningResult object anyway.